packages/javascript/bh-shared-ui/src/views/Explore/ContextMenu/AssetGroupMenuItemPrivilegeZonesEnabled.tsx

```diff
@@ -16,95 +16,149 @@
 
 import {
     Button,
+    Dialog,
     DialogActions,
     DialogContent,
     DialogDescription,
     DialogPortal,
     DialogTitle,
 } from '@bloodhoundenterprise/doodleui';
 import { MenuItem } from '@mui/material';
-import { FC } from 'react';
+import { AssetGroupTagSelectorAutoCertifySeedsOnly, SeedTypeObjectId } from 'js-client-library';
+import { FC, useState } from 'react';
+import { useMutation } from 'react-query';
 import { Link } from 'react-router-dom';
 
-const AssetGroupMenuItem: FC<{
```

Comment:
Majority of these props were values only needed within this component. Refactored so all that data is now sourced within the component. It may be easier to asset this as a whole new file rather than as a diff.